### PR TITLE
Added Revision and Version in <authoring_tool>

### PR DESCRIPTION
**Pull request by mycheckgithub**

### CODE REVIEW

File: COLLADABaseUtils/include/COLLADABUVersionInfo.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#ifndef __COLLADABU_VERSIONINFO_H__                                  
+#define __COLLADABU_VERSIONINFO_H__           
+#include "COLLADAMayaPrerequisites.h"
```
Comment (jerome-jacob) [inline COLLADABaseUtils/include/COLLADABUVersionInfo.h]: Not sure we should include Maya stuff in COLLADABU...

Comment (RemiArnaud) [inline COLLADABaseUtils/include/COLLADABUVersionInfo.h]: nope, should not at all

**Comment by RemiArnaud** [comment]
I modified Jenkins jobs to fail if openCOLLADA has dependencies inside the plugin (Max,Maya) folders.
Let see if the build fails now:

**Comment by RemiArnaud** [comment]
retest this please

**Comment by RemiArnaud** [comment]
test this please

**Comment by RemiArnaud** [comment]
@krunal - 'retest this please' did not work. 'test this please' worked.  do you know why?

File: COLLADAMaya/include/COLLADAMayaSyntax.h [hunk 2]
```diff
@@ -25,6 +25,8 @@ namespace COLLADAMaya
     // COLLADA NetAllied Systems
     static const char* TRANSLATOR_VENDOR                            = "NetAllied Systems GmbH";
     static const char* TRANSLATOR_VERSION                           = "1.3.0";
```
Comment (RemiArnaud) [inline COLLADAMaya/include/COLLADAMayaSyntax.h]: We won't maintain a version per subproject, but a version of the openCOLLADA overall project
There should be a MAJOR and MINOR version in COLLADABUVersionInfo.h, in addition to revision.
The MAJOR will be set to 1
The MINOR will be set to 6

TRANSLATOR_VERSION should be removed in favor of using version numbers from COLLADABU

File: COLLADAMax/src/COLLADAMaxColladaPlugin.cpp [hunk 3]
```diff
@@ -16,10 +16,7 @@ namespace COLLADAMax
 
 	const TCHAR* COLLADAPlugin::PLUGIN_VERSION_TCHAR = __T("Version: 1.4.1");
 	const String COLLADAPlugin::PLUGIN_VERSION_STRING = "Version: 1.4.1";
```
Comment (RemiArnaud) [inline COLLADAMax/src/COLLADAMaxColladaPlugin.cpp]: This is the COLLADA Schema version, not to be mixed up with he version of the plugin.

File: COLLADAMaya/src/COLLADAMayaFileTranslator.cpp [hunk 4]
```diff
@@ -83,12 +84,7 @@
         stream << MAYA_API_VERSION;
 
         COLLADAMaya::String revision ( COLLADAMaya::TRANSLATOR_VERSION );
```
Comment (RemiArnaud) [inline COLLADAMaya/src/COLLADAMayaFileTranslator.cpp]: TRANSLATOR_VERSION need to be replaced (or its definition changed)

**Pull request merged**